include/deal.II/lac/trilinos_epetra_vector.h

```diff
@@ -42,6 +42,9 @@ namespace LinearAlgebra
   template <typename Number>
   class ReadWriteVector;
 
+  /**
+   * A namespace for classes that provide wrappers for Epetra from Trilinos.
```

Comment:
maybe say "...for the classes that Trilinos provides through its Epetra subpackage" (possibly adding URLs if you feel ambitious)